src/Exceptions/Handlers/LegacyDevelopment.php

```diff
@@ -88,4 +69,19 @@ protected function formatStackTrace($stackTrace)
 
         return $return;
     }
+
+    /**
+     * @param mixed $arg
+     * @return string
+     */
+    private function getDisplayNameOfValue(mixed $arg): string
```

Comment:
This function is not covered by tests / excluded

Reply:
Bspw.
```diff
diff --git a/src/Exceptions/Handlers/LegacyDevelopment.php b/src/Exceptions/Handlers/LegacyDevelopment.php
index 8e32e24a..b1c381a1 100644
--- a/src/Exceptions/Handlers/LegacyDevelopment.php
+++ b/src/Exceptions/Handlers/LegacyDevelopment.php
@@ -80,7 +80,7 @@ class LegacyDevelopment extends Legacy
             'string', 'integer', 'double' => (string)$arg,
             'boolean'  => $arg ? 'true' : 'false',
             'object'   => get_class($arg),
-            'resource' => get_resource_type($arg),
+            'resource' => get_resource_type($arg), // @codeCoverageIgnore
             default    => gettype($arg),
         };
     }
diff --git a/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php b/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php
index bd0c6990..42e81bfe 100644
--- a/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php
+++ b/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php
@@ -11,8 +11,9 @@ use PHPUnit\Framework\TestCase;
 class LegacyDevelopmentTest extends TestCase
 {
     /**
-     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::formatStackTrace()
-     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::render()
+     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::formatStackTrace
+     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::render
+     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::getDisplayNameOfValue
      */
     public function testRender()
     {
```

Reply:
added comments